packages/libs/react-ui/src/components/Notification/Notification.css.ts

```diff
@@ -0,0 +1,141 @@
+import { ColorType, sprinkles, vars } from '../../styles';
+
+import {
+  createVar,
+  fallbackVar,
+  globalStyle,
+  style,
+  styleVariants,
+} from '@vanilla-extract/css';
+
+const contrastColor = createVar(),
+  surfaceColor = createVar();
+
+export const containerClass = style([
+  sprinkles({
+    borderWidth: 'md',
+    display: 'flex',
+    justifyContent: 'flex-start',
+    alignItems: 'flex-start',
+    //paddingX: 'md',
+    width: 'max-content',
+    height: 'min-content',
+  }),
+  {
+    border: `1px solid ${fallbackVar(contrastColor, vars.colors.neutral6)}`,
+    color: fallbackVar(contrastColor, vars.colors.neutral6),
+    backgroundColor: fallbackVar(surfaceColor, vars.colors.neutral2),
+    borderLeftWidth: vars.sizes['1'],
+    position: 'relative',
+  },
+]);
+
+export const footerClass = style([
+  sprinkles({
+    marginY: 'md',
+  }),
+]);
+
+export const expandClass = style([
+  sprinkles({}),
+  {
+    width: '100% !important',
+  },
+]);
+
+export const simpleClass = style([
+  sprinkles({
+    borderRadius: 'md',
+    paddingRight: 'xl',
+  }),
+  {
+    borderLeftWidth: 1,
+  },
+]);
+
+export const cardTitleClass = style([
+  sprinkles({
+    marginY: 'sm',
+  }),
+]);
+
+export const contentClass = style([
+  sprinkles({
+    marginBottom: 'sm',
+  }),
+]);
+
+export const simpleContentClass = style([
+  sprinkles({
+    marginY: 'md',
+  }),
+]);
+
+export const colorVariants = styleVariants(
+  {
+    default: 'default',
+    primary: 'primary',
+    secondary: 'secondary',
+    positive: 'positive',
+    warning: 'warning',
+    negative: 'negative',
+  },
+  (color) => {
+    return [
+      containerClass,
+      {
+        vars: {
+          [contrastColor]:
+            color === 'default'
+              ? vars.colors.neutral6
+              : vars.colors[`${color as ColorType}Contrast`],
+          [surfaceColor]:
+            color === 'default'
+              ? vars.colors.neutral2
+              : vars.colors[`${color as ColorType}Surface`],
+        },
+      },
+    ];
+  },
+);
```

Comment:
Reminders: 
- Use sprinkles where you can! So for color and background color you can use sprinkles
- Type the colors when creating the object so that you don't have to cast them to a specific type later https://dev.to/ylerjen/typescript-cast-is-a-type-breaker-1jbh